packages/app-store/around/static/icon.svg

```diff
@@ -0,0 +1,15 @@
+<svg width="326" height="218" viewBox="0 0 326 218" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I don't know if this is supposed to look like this:
![image](https://user-images.githubusercontent.com/3504472/174666941-5d6fdfe2-138f-431d-b443-ef513ef6d97b.png)